### PR TITLE
Make UARTService characteristics be at least as large as max MTU-3

### LINKED ISSUE #94: Receiving >20 bytes at once from Bluefruit Connect app not working
@ Thornhill in discord says a UART message of  >20 bytes from the Bluefruit Connect app to, say, `ble_uart_echo_test`, is dropped. Sending to/from two CircuitPython boards does work. Also, does work sending to Arduino.

Confirmed by @caternuson.

### PR DESCRIPTION
**Pull request by dhalbert**
Fixes #94.

Nordic UART Service clients expect that they will be able to send MTU-3 bytes all at once, in a single write. The current code was expecting no more than 20-byte writes.

Currently the maximum MTU-3 used by all CircuitPython ports is 512-3 (which is wrong, it should be 512). Change the `max_length` for the characteristic to be 512, and also enlarge the PacketBuffer correspondingly.

Tested with UART sends from Adafruit Bluefruit Connect app and nRF Toolbox on iOS on nRF52840 and Metro ESP32-S3. Also tested for regression with an nRF52840 uART client.

### CODE REVIEW

**Comment by dhalbert** [comment]
User Thinman in discord tested and it fixes their problem.

File: adafruit_ble/characteristics/__init__.py [hunk 1]
```diff
@@ -10,6 +10,8 @@
 
 from __future__ import annotations
 
+# pylint: disable=too-many-arguments
```
Comment (tannewt) [inline adafruit_ble/characteristics/__init__.py:13]: Is there a central place this can be disabled?

Comment (dhalbert) [reply adafruit_ble/characteristics/__init__.py]: In another I PR, I changed `max-args=` in `.pylintrc`, which makes more sense for this kind of override.

**Review by tannewt — APPROVED** [review]
Thanks for the fix!

**Pull request merged**